src/lists.py

```diff
@@ -42,17 +42,20 @@ def print_list(x):
 match sys.argv[1]:
     case "mean":
         # put your solution to the first exercise here
-        mean = "mean of x"
+        mean = sum(x)/len(x) #"mean of x"
         print(mean)
 
     case "times":
         # Put your solution to the second exercise here
-        times_three = []
+        times_three = x*3 
```

Comment:
If `x` is a list, then `x*3` repeats the list three times.

```python
>>> x = [1, 2, 3]
>>> x * 3
[1, 2, 3, 1, 2, 3, 1, 2, 3]
```

It doesn't multiply each entry by `3` (unlike for example in R).

You need to explicitly multiply each entry.

```python
>>> [3 * a for a in x]
[3, 6, 9]
```

Reply:
Thank you!